packages/amplify_datastore/example/lib/models/BelongsToModel.dart

```diff
@@ -95,15 +109,30 @@ class BelongsToModel extends Model {
     buffer.write("BelongsToModel {");
     buffer.write("id=" + "$id" + ", ");
     buffer.write("name=" + "$_name" + ", ");
-    buffer.write("child=" + (_child != null ? _child!.toString() : "null"));
+    buffer.write(
+        "child=" + (_child != null ? _child!.toString() : "null") + ", ");
+    buffer.write("createdAt=" +
+        (_createdAt != null ? _createdAt!.format() : "null") +
+        ", ");
+    buffer.write(
+        "updatedAt=" + (_updatedAt != null ? _updatedAt!.format() : "null"));
     buffer.write("}");
 
     return buffer.toString();
   }
 
-  BelongsToModel copyWith({String? id, String? name, ChildModel? child}) {
-    return BelongsToModel(
-        id: id ?? this.id, name: name ?? this.name, child: child ?? this.child);
+  BelongsToModel copyWith(
+      {String? id,
+      String? name,
+      ChildModel? child,
+      TemporalDateTime? createdAt,
+      TemporalDateTime? updatedAt}) {
```

Comment:
As `createdAt` and `updatedAt` are readonly, I think they are not supposed to be parameters of the `copyWith` method, as a user can pass in any value to override the original value.

Reply:
Thanks good point.  I asked Aaron to update the amplify codegen to not have these parameters in the copyWith method anymore